Rakefile

```diff
@@ -14,6 +14,7 @@ task :test do
       '/doc/reports/translations',
       '/doc/reports/vimpatch',
       '/doc/user',
+      '#',
```

Comment:
The "About" link has `href="#"`. This will cause warnings a la `# is not a valid anchor on this page` in HTML-Proofer (`bundle exec rake`/Travis build).